### PR TITLE
feat: add `version` to payload of excel export API

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Please see order by commits

### Things to Talk About
Since the console API has not been deployed yet, you will need to **run local server to test it.**🥲

### CODE REVIEW

File: apps/web/src/services/cost-explorer/budget/budget-main/modules/budget-list/BudgetList.vue [hunk 1]
```diff
@@ -53,21 +52,21 @@ const state = reactive({
...
-    }),
 });
 
 const fetchBudgetUsages = async (): Promise<{more: boolean; results: BudgetUsageAnalyzeResult[]}> => {
     try {
         state.loading = true;
-        return await SpaceConnector.clientV2.costAnalysis.budgetUsage.analyze(state.budgetUsageParams);
+        budgetUsageApiQueryHelper
+            .setFilters(state.queryStoreFilters)
+            .setPage(state.pageStart, state.pageLimit);
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/budget/budget-main/modules/budget-list/BudgetList.vue:62]: question: Why is this moved into here instead of computed hook? Isn't it inefficient because of recalculation on every list api?

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue [hunk 2]
```diff
@@ -306,21 +310,18 @@ const handleChange = async (options: any = {}) => {
     if (costAnalysisPageState.period) tableState.items = getRefinedChartTableData(results, costAnalysisPageState.granularity, costAnalysisPageState.period);
     tableState.more = more;
 };
+const costAnalyzeExportApiQueryHelper = new ApiQueryHelper();
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue:313]: nitpick: Please use QueryHelper instead of ApiQueryHelper in case that you don't need any other options like pagination or sort. 
It's pretty risky though...

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue [hunk 3]
```diff
@@ -306,21 +310,18 @@ const handleChange = async (options: any = {}) => {
...
-                granularity: costAnalysisPageState.granularity,
-                group_by: costAnalysisPageState.groupBy,
-                start: dayjs.utc(costAnalysisPageState?.period?.start).format(dateFormat),
-                end: dayjs.utc(costAnalysisPageState?.period?.end).format(dateFormat),
-                filter: analyzeApiQueryHelper.data.filter,
-                query: analyzeApiQueryHelper.data,
+                data_source_id: costAnalysisPageStore.selectedDataSourceId,
+                query: {
+                    ...state.analyzeQuery,
+                    ...costAnalyzeExportApiQueryHelper.data,
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue:323]: nitpick: Please check if `costAnalyzeExportApiQueryHelper.data` doesn't override or add anything like sort or pagination.

File: apps/web/src/services/cost-explorer/budget/budget-main/modules/budget-list/BudgetList.vue [hunk 4]
```diff
@@ -117,14 +117,19 @@ const handleExport = async () => {
...
+    budgetUsageExportApiQueryHelper.setFilters(state.queryStoreFilters);
+    const _query = getBudgetUsageAnalyzeRequestQuery(state.sort, state.period);
     await store.dispatch('file/downloadExcel', {
         url: '/cost-analysis/budget-usage/analyze',
         param: {
-            ...state.budgetUsageParams,
-            query: budgetUsageApiQueryHelper.data,
+            query: {
+                ..._query,
+                ...budgetUsageExportApiQueryHelper.data,
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/budget/budget-main/modules/budget-list/BudgetList.vue:127]: nitpick: I think it would be safer to change the order of those two queries.

**Review by WANZARGEN — APPROVED** [review]
I've reviewed the code, and I appreciate your work! I've added a few minor comments, so please take a moment to review them when you have the chance.

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue (hunk 3 above)
Comment (yuda110) [reply apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue:323]: I checked it!

File: apps/web/src/services/cost-explorer/budget/budget-main/modules/budget-list/BudgetList.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/cost-explorer/budget/budget-main/modules/budget-list/BudgetList.vue:62]: I'll move it back to computed 😅
